### PR TITLE
Improved renderer + info

### PR DESCRIPTION
**Pull request by OryJonay**

### CODE REVIEW

File: oddsgym/envs/base.py [hunk 1]
```diff
@@ -193,7 +194,7 @@ def render(self, mode='human'):
         msg : str
             A string with the current balance and the current step.
         """
-        return 'Current balance at step {}: {}'.format(self.current_step, self.balance)
+        return print('Current balance at step {}: {}'.format(self.current_step, self.balance))
```
Comment (OryJonay) [inline oddsgym/envs/base.py]: This line is no longer tested in the unit tests (due to the removal of the test).
Also, the ThreeWaySoccer envs also implement a render method (overrides and uses this method). Need to test that this still works.

File: oddsgym/envs/daily_bets.py [hunk 2]
```diff
@@ -43,6 +43,8 @@ class DailyOddsEnv(BaseOddsEnv):
 
     """
 
+    headers = ['Date', 'Current Step', 'Odds', 'Verbose Action', 'Action', 'Balance', 'Reward', 'Results', 'Done']
```
Comment (OryJonay) [inline oddsgym/envs/daily_bets.py]:
```suggestion
    HEADERS = ['Date', 'Current Step', 'Odds', 'Verbose Action', 'Action', 'Balance', 'Reward', 'Results', 'Done']
```

File: oddsgym/envs/base.py [hunk 3]
```diff
@@ -52,6 +53,7 @@ class BaseOddsEnv(gym.Env):
     """
 
     metadata = {'render.modes': ['human']}
+    headers = ['Current Step', 'Odds', 'Verbose Action', 'Action', 'Balance', 'Reward', 'Results', 'Done']
```
Comment (OryJonay) [inline oddsgym/envs/base.py]:
```suggestion
    HEADERS = ['Current Step', 'Odds', 'Verbose Action', 'Action', 'Balance', 'Reward', 'Results', 'Done']
```

File: tests/test_daily_env.py [hunk 4]
```diff
@@ -203,3 +203,18 @@ def test_get_odds_in_observation_space(daily_bets_env, current_step_value):
...
+    assert info['current_step'] == 0
+    numpy.testing.assert_array_equal(info['odds'], numpy.array([[1., 2.], [4., 3.]]))
+    numpy.testing.assert_array_equal(info['verbose_action'], numpy.array([[['w', 'l']], [['w', 'l']]]))
+    numpy.testing.assert_array_equal(info['action'], numpy.array([_(0), _(0)]))
+    assert info['balance'] == 10
+    assert info['reward'] == 0
+    assert not info['legal_bet']
+    assert info['results'] is None
+    assert not info['done']
+    daily_bets_env.pretty_print_info(info)
```
Comment (OryJonay) [inline tests/test_daily_env.py]:
```suggestion
    daily_bets_env.pretty_print_info(info)
    
```
(add newline at end of file)

File: oddsgym/envs/base.py (hunk 1 above)
Comment (dgingi) [reply oddsgym/envs/base.py]: fixed at ThreeWaySoccer. 
There is not test for it, only in the base env, but it was changed to test_info.
should we actually check if it prints to sout?

File: oddsgym/envs/daily_bets.py (hunk 2 above)
Comment (dgingi) [reply oddsgym/envs/daily_bets.py]: 👍

File: oddsgym/envs/base.py (hunk 3 above)
Comment (dgingi) [reply oddsgym/envs/base.py]: 👍

File: tests/test_daily_env.py (hunk 4 above)
Comment (dgingi) [reply tests/test_daily_env.py]: 👍

File: oddsgym/envs/soccer.py [hunk 5]
```diff
@@ -37,9 +37,9 @@ def render(self, mode='human'):  # pragma: no cover
         msg : str
             A string with the current team names, balance and step.
         """
-        return 'Home Team {} VS Away Team {}. {}'.format(self.teams[self.current_step % self._odds.shape[0]][0],
+        print('Home Team {} VS Away Team {}. {}'.format(self.teams[self.current_step % self._odds.shape[0]][0],
                                                          self.teams[self.current_step % self._odds.shape[0]][1],
-                                                         super().render(mode))
+                                                         super().render(mode)))
```
Comment (OryJonay) [inline oddsgym/envs/soccer.py]: Does this work? After the change, render returns None, so it won't add the current step & balance when rendering.

File: oddsgym/envs/soccer.py [hunk 6]
```diff
@@ -91,7 +91,7 @@ def render(self, mode='human'):  # pragma: no cover
         teams_str = ', '.join(['Home Team {} VS Away Team {}'.format(row.home_team, row.away_team)
                                for row in self.teams.iloc[index].itertuples()])
         balance_str = super().render(mode)
-        return '. '.join((teams_str, balance_str))
+        print('. '.join((teams_str, balance_str)))
```
Comment (OryJonay) [inline oddsgym/envs/soccer.py]: Same like previous comment. Also, why is there a different implementation for the Percentage env and the regular env? (why does the Percentage env even implements a render method, it should have been the same one like the regular ThreeWaySoccer env)

File: oddsgym/envs/base.py (hunk 1 above)
Comment (OryJonay) [reply oddsgym/envs/base.py]: Yeah, the way (that I know) to do it is to "redirect" stdout when testing to a temporary file and then just test that the expected string is in the file. Read pytest documentation on the subject: https://docs.pytest.org/en/stable/capture.html#accessing-captured-output-from-a-test-function

File: oddsgym/envs/soccer.py (hunk 5 above)
Comment (dgingi) [reply oddsgym/envs/soccer.py]: fixed

File: oddsgym/envs/soccer.py (hunk 6 above)
Comment (dgingi) [reply oddsgym/envs/soccer.py]: the two classes that have render are:
ThreeWaySoccerOddsEnv and ThreeWaySoccerDailyOddsEnv (first print one game, the second prints a list of games)
Percentage env doesn't have it is own implementation

**Pull request merged**